src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.DukeException;
+import duke.storage.TaskList;
+import duke.task.Task;
+
+/**
+ * Represents a command to add new task
+ */
+public class AddCommand implements Command {
```

Comment:
(Also a general comment for your other command classes:)

I understand that by `AddCommand` you are referring to "a command by the user to add some task"; perhaps consider using something like `AddTaskCommand` as the class name instead to make it clear that you are specifying "some class to represent commands to add tasks" instead of "some class to add some commands".